src/hooks/useUploadsManager/useUploadsManager.tsx

```diff
@@ -22,62 +31,195 @@ export const UploadManagerProvider: React.FC = ({ children }) => {
   const { displaySnackbar } = useSnackbar()
   const updateAsset = useUploadsStore((state) => state.updateAsset)
 
-  const uploadsStateIds = useUploadsStore(
-    (state) =>
-      state.uploadsState.filter((item) => item.parentObject?.type === 'video').map((item) => item.parentObject.id),
-    shallow
-  )
   const channelUploadsState = useUploadsStore(
     (state) => state.uploadsState.filter((asset) => asset.owner === activeChannelId),
     (prevState, newState) => isEqual(prevState, newState)
   )
+
   // \/ workaround for now to not show completed uploads but not delete them since we may want to show history of uploads in the future
   const [ignoredAssetsIds, setIgnoredAssetsIds] = useState<string[]>([])
-  const { loading: channelLoading } = useChannel(activeChannelId ?? '')
-  const { loading: videosLoading } = useVideos(
-    {
-      where: {
-        id_in: uploadsStateIds,
-      },
-    },
-    { skip: !uploadsStateIds.length }
-  )
+  const [queryNodePendingAssets, setQueryNodePendingAssets] = useState<AssetUpload[]>([])
```

Comment:
Why not make this part of the store?

Reply:
I thought we don't want to make query node data part of the store. Can make that though - that will require some additional usage of `updateAsset` on initial sync up

Reply:
Ah okay I misunderstood what this is. Let me think about this